src/components/DictionaryView.vue

```diff
@@ -1,29 +1,35 @@
 <template>
   <div>
-    <h1 class="text-3xl text-gray-700">{{ word }}</h1>
-    <h3 class="text-2xl text-gray-700">Definitions</h3>
-    <ul>
-      <li
-        v-for="(definition, index) in definitions"
-        :key="index"
-        class="text-gray-700"
-      >
-        {{ definition.definition }} 
-      </li>
-    </ul>
+    <v-progress-linear
+      indeterminate
```

Comment:
help me understand this code!

Reply:
It's just a vuetify component. When it has the indeterminate prop, it's just a spinning circle.